docs/src/Command/GenerateReferenceCommand.php

```diff
@@ -0,0 +1,136 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace PDG\Command;
+
+use PDG\Services\Reference\PhpDocHelper;
+use PDG\Services\Reference\ReflectionHelper;
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Filesystem\Path;
+
+#[AsCommand(name: 'pdg:generate-reference')]
+class GenerateReferenceCommand extends Command
+{
+    private readonly array $config;
+    private string $root;
+    private \ReflectionClass $reflectionClass;
+
+    public function __construct(
+        private readonly PhpDocHelper $phpDocHelper,
+        private readonly ReflectionHelper $reflectionHelper,
+        string $name = null
+    ) {
+        parent::__construct($name);
+        $this->config = (require 'src/config.php')();
```

Comment:
At some point we should put this in a bundle using a symfony Configuration?

Reply:
Yeah it is actually troublesome now that I'm trying to add tests for the commands. (Paths do not match etc ...)

What do you think would be the best way to handle this ?